src/Sylius/Behat/Element/Admin/Product/ProductAttributesFormElement.php

```diff
@@ -0,0 +1,197 @@
+    {
+        $attribute = $this->getDocument()->find('css', sprintf('.attribute .attribute-label:contains("%s")', $attributeName));
+
+        return
+            $attribute->getParent()->getParent()->find('css', '.attribute-input input')->getValue() === $value &&
+            $attribute->find('css', '.globe.icon') !== null
+        ;
+    }
+
+    public function addNonTranslatableAttribute(string $attributeName, string $value): void
+    {
+        $this->clickTabIfItsNotActive();
+
+        $attributeOption = $this->getElement('attributes_choice')->find('css', sprintf('option:contains("%s")', $attributeName));
+        $this->selectElementFromAttributesDropdown($attributeOption->getAttribute('value'));
+
+        $this->getDocument()->pressButton('Add attributes');
+        $this->waitForFormElement();
+
+        $this->getElement('non_translatable_attribute_value', ['%attributeName%' => $attributeName])->setValue($value);
+    }
+
+    public function addAttribute(string $attributeName): void
+    {
+        $this->changeTab();
+        $this->selectAttributeToBeAdded($attributeName);
+        $this->clickButton('Add');
+
+        $this->waitForFormUpdate();
+    }
+
+    public function updateAttribute(string $attributeName, string $value, string $localeCode): void
+    {
+        $this->changeTab();
+        $this->changeAttributeTab($attributeName);
+
+        $attributeValue = $this->getElement('attribute_value', ['%attributeName%' => $attributeName, '%localeCode%' => $localeCode]);
+
+        match ($attributeValue->getTagName()) {
+            'input' => $attributeValue->setValue($value),
+            'select' => $attributeValue->selectOption($value),
+            default => throw new \InvalidArgumentException('Unsupported attribute value type'),
+        };
+
+        $attributeValue->blur();
+        $this->waitForFormUpdate();
+    }
+
+    public function getAttributeValue(string $attribute, string $localeCode): string
+    {
+        $this->changeTab();
+        $this->changeAttributeTab($attribute);
+
+        $attributeValue = $this->getElement('attribute_value', ['%attributeName%' => $attribute, '%localeCode%' => $localeCode]);
+
+        return match ($attributeValue->getTagName()) {
+            'input' => $attributeValue->getValue(),
+            'select' => $attributeValue->getText(),
+            default => throw new \InvalidArgumentException('Unsupported attribute value type'),
+        };
+    }
+
+    public function addSelectedAttributes(): void
+    {
+        $this->changeTab();
+        $this->clickButton('Add');
+        $this->waitForFormUpdate();
+    }
+
+    public function getNumberOfAttributes(): int
+    {
+        return count($this->getDocument()->findAll('css', '[data-test-product-attribute-tab]'));
+    }
+
+    protected function getDefinedElements(): array
+    {
+        return [
+            'product_attribute_delete_button' => '[data-test-product-attribute-delete-button="%attributeName%"]',
```

Comment:
We should use snake_case notation for these parameters, but it could be adjusted in a separate PR for all pages and elements 🖖🏻 
```suggestion
            'product_attribute_delete_button' => '[data-test-product-attribute-delete-button="%attribute_name%"]',
```